catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -401,14 +392,10 @@ public void testGetResourceInfoWithEscapedCharacters() throws Exception {
     ResourceOperations.ResourceInfo resourceInfo =
         resourceOperationsSpy.getResourceInfo(
             resourceRequestMock, "site", isEnterprise, federatedSite, requestProperties, false);
-    verify(resourceOperationsSpy)
-        .createPropertyStartsWithQuery(
-            Metacard.RESOURCE_URI,
-            "scheme:///" + URLEncoder.encode("part1?file://some/pathto/file"));
     assertThat(resourceInfo.getResourceUri(), equalTo(uri));
   }
 
-  @Test
+  @Test(expected = ResourceNotFoundException.class)
   public void testGetResourceInfoWithFragmentAndMultipleResults() throws Exception {
```

Comment:
❓ Why does this unit test result in a `ResourceNotFoundException`?

Reply:
Before if we got more than one resource back we would just take the first one in the array. Now, We want to throw an exception so this test will make sure that happens.